app/templates/index.hbs

```diff
@@ -60,7 +60,7 @@
   <div local-class='lists' data-test-lists>
     <section data-test-new-crates>
       <h2>New Crates</h2>
-      <ol local-class="list" aria-busy="{{this.dataTask.isRunning}}">
+      <ol local-class="list" aria-busy={{this.dataTask.isRunning}}>
```

Comment:
we were explicitly wrapping the values in quotes here to cast them into a string because we want `aria-busy="false"` instead of removing the attribute when the task is not running